x-pack/plugins/profiling/server/routes/flamechart.ts

```diff
@@ -26,6 +26,7 @@ export function registerFlameChartSearchRoute({
   router.get(
     {
       path: paths.Flamechart,
+      options: { tags: ['access:profiling'] },
```

Comment:
__note:__ for after feature-freeze, it'd be great if you can add functional/api-integration tests that ensure that only users with specific privilege can access your routes.